style.css

```diff
@@ -0,0 +1,866 @@
+    flex-direction: column;
+    justify-self: center;
+    align-items: flex-start;
+    gap: 10px;
+    width: 55%;
+  }
+
+  .domaine {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    gap: 200px;
+    width: 100%;
+  }
+
+  .domaine h2 {
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    color: #344563;
+    margin-bottom: 0;
+  }
+
+  .dom2,
+  .dom3 {
+    padding-bottom: 20px;
+    border-bottom: 1px solid #dfe1e6;
+  }
+
+  .list_language {
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+    width: 324px;
+  }
+
+  .list_language h3 {
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    color: #253858;
+  }
+
+  .lang img {
+    width: 48px;
+    height: 48px;
+  }
+
+  .lang {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 112px;
+    height: 110px;
+    background: #f7f7f9;
+    border-radius: 8px;
+  }
+
+  /* Contact css start */
+
+  .contact {
+    background-color: #6070ff;
+    background-image: url(./images/contact_background.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid #6070ff;
+    border-top-left-radius: 80px;
+    margin-top: -200px;
+    padding-bottom: 20px;
+    gap: 10px;
+  }
+
+  .contact h2 {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #fff;
+    margin: 80px 0 0 0;
+  }
+
+  .contact p {
+    height: 102px;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+    width: 55%;
+  }
+
+  form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 15px;
+  }
+
+  input {
+    width: 447px;
+    height: 44px;
+    border: 1px solid #cfd8dc;
+    border-radius: 2px;
+    background: white;
+    font-weight: 100;
+    font-size: 20px;
+    line-height: 20px;
+    color: #172b4d;
+  }
+
+  input::placeholder {
+    color: #172b4d;
+    font-size: 15px;
+    padding-left: 10px;
+  }
+
+  textarea {
+    width: 447px;
+    height: 170px;
+    background: white;
+    border: 1px solid #cfd8dc;
+    border-radius: 2px;
+    margin-bottom: 0;
+    text-align: start;
+    font-size: 20px;
+  }
+
+  textarea::placeholder {
+    position: absolute;
+    color: #b3b3b3;
+    font-family: 'Poppins', sans-serif;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    padding: 10px;
+  }
+
+  .contact button {
+    margin-left: 150px;
+  }
+}
+
+/* Mobile part */
+
+@media (max-width: 767px) {
+  .desk_work,
+  .nav_links a,
+  .desk_popup,
+  .popupd {
+    display: none;
+  }
+
+  /* header section css */
+
+  nav i {
+    font-size: 30px;
+    color: #6070ff;
+    padding-right: 40px;
```

Comment:
[OPTIONAL] The project was supposed to be done with a mobile-first approach from the beginning and if you are sticking with that, this media query will not be necessary since all styling will be done outside and on the desktop, you will manipulate specific properties and not all. Using this approach will make your CSS file less cumbersome and less repetitive.

Reply:
right